Header header

string name
string type
string status
string contract
geometry_msgs/PoseWithCovariance pose
 